java-frontend/src/main/java/org/sonar/java/resolve/TypeAndReferenceSolver.java

```diff
@@ -427,7 +434,15 @@ public void visitThrowStatement(ThrowStatementTree tree) {
   public void visitLambdaExpression(LambdaExpressionTree tree) {
     LambdaExpressionTreeImpl lambdaExpressionTree = (LambdaExpressionTreeImpl) tree;
     if(lambdaExpressionTree.isTypeSet()) {
-      //FIXME Based on set type get correct type of lambda var if required.
+      // type should be tied to a SAM interface
+      List<JavaType> samMethodArgs = resolve.findSamMethodArgs(lambdaExpressionTree.symbolType());
+        for (int i = 0; i < samMethodArgs.size(); i++) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)